api-tests/dev_apis/internal_trusted_storage/test_s004/test_s004.c

```diff
@@ -71,14 +71,14 @@ int32_t psa_sst_get_data_check(security_t caller)
     TEST_ASSERT_MEMCMP(read_buff, write_buff, TEST_BUFF_SIZE/2, TEST_CHECKPOINT_NUM(8));
 
     /* Check we should not be able to access old set data */
-    val->print(PRINT_TEST, "[Check 2] When get api is called for uid whose length in reduced,"
-                            " check old data is inaccessible\n", 0);
+    val->print(PRINT_TEST, "[Check 2] When get API is called for UID whose length in reduced,\n"
```

Comment:
Too long statement, Can we short this?

For example - old data accessibility check for short length

Reply:
review comments taken care